src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          alt="Logo Moyo"
+          src="images/logo.png"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__menu menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="Imac"
+          class="card__img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="card__rate-stars stars">
```

Comment:
You could have declared this .stars block in a similar way as you have done in Stars task.
In the way that is implemented there, there is a modifier on stars which shows the adequate amount of active and inactive stars, as opposed to what you've made here where the styling is fixed